assets/queries/openAPI/invalid_tag_external_documentation_url/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	url := doc.tags[x].externalDocs.url
+	name := doc.tags[x].name
+	not openapi_lib.is_valid_url(url)
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("tags.name=%s", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("tags[%d].externalDocs.url has a valid URL", [x]),
+		"keyActualValue": sprintf("tags[%d].externalDocs.url has an invalid URL", [x]),
+	}
+}
```

Comment:
Tags can also be a field of  a Operation object: https://swagger.io/specification/#operation-object
This query is only checking tags inside OpenAPI Object.

Reply:
In the Operation object, the field 'tags' is a list of strings. There is no access to 'externalDocs'